src/routes/(app)/pos/touch/+page.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+	import './touch.css';
+</script>
+
+<svelte:head>
+	<meta name="viewport" content="width=1024" />
+</svelte:head>
+<main class="fixed top-0 bottom-0 right-0 left-0 bg-white p-4">
```

Comment:
This should be in a `+layout.svelte` - and maybe the layout reset can be in a `+layout.server.ts`

But we can handle this in a follow up PR, just keep it in mind when you start adding other pages - to create a `+layout.svelte`/`+layout.server.ts`